File: C Projects/Basic_Calculator.c
#include<stdio.h>
int main()
{
    int a,b,S,ch;
    printf("Enter the first number:");
    scanf("%d",&a);
    printf("Enter the second number:");
    scanf("%d",&b);
    S=a+b;
    printf("Enter 1 for Addition\nEnter 2 for Subtraction\nEnter 3 for Multiplication\nEnter 4 for Division");
    printf("Enter your choice:");
    scanf("%d",&ch);
    switch(ch)
    {
        case 1: S=a+b;
                printf("Addition of %d and %d is %d",a,b,S);
                break;
        case 2: S=a-b;
                printf("Subtraction of %d and %d is %d",a,b,S);
                break;
        case 3: S=a*b;
                printf("Multiplication of %d and %d is %d",a,b,S);
                break;
        case 4: S=a/b;
                printf("Division of %d and %d is %d",a,b,S);
                break;
        default: printf("Enter correct choice");
    }
}

File: Python/Basic Calculator.py
# Program to create a basic calculator
a = float(input("Enter the first number:"))
b = float(input("Enter the second number:"))
c = int(input("1 for Addition\n2 for Subtraction\n3 for Multiplication\n4 for Division\n5 for Floor Division\n6 for Exponentiation\n7 for Modulus\nEnter your choice:"))
if c == 1:
    sum = a + b
elif c == 2:
    sum = a - b
elif c == 3:
    sum = a * b
elif c == 4:
    sum = a / b
elif c == 5:
    sum = a // b
elif c == 6:
    sum = a ** b
elif c == 7:
    sum = a % b
else:
    print("You have entered wrong input")
    exit()
print("The answer is", sum)